recipe/rsync.php

```diff
@@ -137,8 +137,8 @@
     $server = $server->getConfiguration();
     $host = $server->getHost();
     $port = $server->getPort() ? ' -p' . $server->getPort() : '';
-    $identityMethod = $server->getPemFile() ? 'getPemFile' : $server->getPrivateKey() ? 'getPrivateKey' : false;
-    $identityFile = $identityMethod ? ' -i ' . $server->$identityMethod() : '';
+    $identityMethod = $server->getPemFile() ?: ($server->getPrivateKey() ?: false);
```

Comment:
I am a DevOps and Java guy and don't have much experience with PHP. I was thinking of using if/else but then I thought why not make it similar to how it was that is why I did it like this. I am using php 5.6 on Amazon Linux, it is working on that.